indico/queries/datasets.py

```diff
@@ -262,7 +262,27 @@ def requests(self):
                 debouncer.backoff()
         yield GetDataset(id=dataset_id)
 
+       
+class RemoveFile(GraphQLRequest):
```

Comment:
Could we call this RemoveDatasetFile/something to indicate it is removing from a dataset?

Reply:
Yup! I went with the name for consistency with AddFiles, but what I can do is add an alias for AddFiles to match this naming convention